lib/src/test/java/vmath/Mat2Test.java

```diff
@@ -9,6 +9,26 @@ public class Mat2Test {
 
     @Test
     void givenNoArgs_whenMat2Created_thenMatrixIsIdentity() {
+        Mat2 matrix = new Mat2();
+
+        assertEquals(1.0f, matrix.get(0, 0));
+        assertEquals(0.0f, matrix.get(0, 1));
+        assertEquals(0.0f, matrix.get(1, 0));
+        assertEquals(1.0f, matrix.get(1, 1));
+    }
+
+    @Test
+    void givenIndices_whenGetCalled_thenReturnValueMatches() {
         //TODO
     }
-}
\ No newline at end of file
+
+    @Test
+    void givenInvalidIndices_whenGetCalled_thenThrowIllegalArgumentException() {
+        Mat2 matrix = new Mat2();
+
+        assertThrows(IndexOutOfBoundsException.class, () -> matrix.get(6, 1));
```

Comment:
Please adjust the name of this test: it checks for whether get() throws IndexOutOfBoundsException, not IllegalArgumentException.

Reply:
I renamed this as requested